custom/static/js/pixelmap.js

```diff
@@ -26,24 +26,8 @@ Pixelmap.prototype.loadImage = function (imageId) {
     /* Load an image for display or annotation use. */
     var loaded = $.Deferred();
 
-    // TODO: do Ajax via Backbone or Angular
-    function getCookie(cname) {
-        var name = cname + '=';
-        var ca = document.cookie.split(';');
-        for (var i = 0; i < ca.length; i++) {
-            var c = ca[i];
-            while (c.charAt(0) === ' ') {
-                c = c.substring(1);
-            }
-            if (c.indexOf(name) === 0) {
-                return c.substring(name.length, c.length);
-            }
-        }
-        return '';
-    }
     $.ajax({
-        url: '/api/v1/item/' + imageId + '/tiles',
-        headers: {'Girder-Token': getCookie('girderToken')}
```

Comment:
Why is the `Girder-Token` cookie no longer necessary? In a local test I get 400 errors loading the images when viewing this page.